packages/dds/tree/src/simple-tree/schemaTypes.ts

```diff
@@ -463,6 +586,87 @@ export function normalizeAllowedTypes(
 	return normalized;
 }
 
+/**
+ * Normalizes an allowed type to an {@link AnnotatedAllowedType}, by adding empty annotations if they don't already exist.
+ *
+ * @internal
```

Comment:
Is there a use-case for other packages in our repo calling these? If not, don't mark them as internal.